### PR TITLE
[SB/MRH][76549318] trigger new event when clearing errors

### PR DESCRIPTION
**Pull request by sbrauer**
Initial use case is for AG feature flip story for handling
register/login errors differently in the UI.
https://www.pivotaltracker.com/story/show/76549318

### CODE REVIEW

File: src/login.coffee [hunk 1]
```diff
@@ -279,6 +279,7 @@ define ['jquery', 'primedia_events', 'jquery.cookie'], ($, events) ->
     _clearErrors: ($div) ->
       $div.find('form p').removeClass('error')
       $div.find('.errors').empty()
+      events.trigger('event/clearLoginErrors')
```
Comment (c0) [inline src/login.coffee]: What do you think about using the past tense `event/loginErrorsCleared`?

**Comment by sbrauer** [comment]
@c0 - Please don't merge yet... may need change depending on new info from Yong.

**Comment by sbrauer** [comment]
@c0 - This is ready for another review.
Made it possible to disable prefilling of the email inputs based on zid email.

**Comment by sbrauer** [comment]
oh wait...

**Comment by sbrauer** [comment]
Think we're good to go again.  Please review and merge.

**Comment by c0** [comment]
:+1:

**Pull request merged**